scrapers/sis_scraper/main.py

```diff
@@ -211,14 +217,16 @@ async def get_classes_with_code(term, code):
         return await request.text()
 
 
-async def scrape_subject(term, name, code):
+async def scrape_subject(term, name, code, registration_dates):
```

Comment:
Why is this using an out-param instead of returning two values?

Reply:
Changed in latest commit to use multiple return values.